Gemfile

```diff
@@ -124,3 +125,6 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
+
+# To avoid polling for changes:
+# gem 'wdm', '>= 0.1.0' if Gem.win_platform?
```

Comment:
Should we uncomment this line? What does `wdm` do?

Reply:
It is not important. I got this error while setting up the database. Turned out it was just a warning/suggestion. Forgot to delete it afterwards.

Reply:
👍